src/todo.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable quotes */
+const InnerTodo = document.querySelector(".todo-inner");
+
+export default class display {
+  static getTodo = () => {
+    let Todo;
+    if (localStorage.getItem("todo") == null) {
+      Todo = [];
+    } else {
+      Todo = JSON.parse(localStorage.getItem("todo"));
+    }
+    return Todo;
+  };
+
+  static addRemoveEvent = () => {
+    const trash = document.querySelectorAll(".trash");
+    trash.forEach((task, i) => {
+      task.addEventListener("click", (ev) => {
+        ev.preventDefault();
+        this.removeTodo(i);
+      });
+    });
+  };
+
+  static removeTodo = (id) => {
+    const toDos = display.getTodo();
+    toDos.splice(id, 1);
+    localStorage.setItem("todo", JSON.stringify(toDos));
+    this.loadTodo(toDos);
+  };
+
+  static loadTodo = (item) => {
+    let display = " ";
+    item.forEach((elem, i) => {
+      display += `
+      <div class="todo-check flex">
+      <div class="checkbox">
```

Comment:
- [ ] as mentioned by the requirements, kindly add the functionality of editing task descriptions, the adding and removing are working great only the updating function is missing 🙏